doc/rst/technotes/debuggingChapel.rst

```diff
@@ -0,0 +1,66 @@
+.. _readme-debugging-chapel:
+
+======================================================
+Debugging Chapel Source Code: Best Known Configuration
+======================================================
+
+The Chapel compiler provides a number of mechanisms to make the generated code
+more readable in a debugger. There is limited support available for the LLVM
+backend so it is recommended to use the C backend. 
+
+In Summary
+==========
+
+1) Build the compiler with ``CHPL_TARGET_COMPILER=gnu make``.
```

Comment:
This threw me off as an assignment to `CHPL_TARGET_COMPILER` of the value `gnu make`. Perhaps this could be worded in a way that first mentions the environment variable, and _then_ provides the build command? e.g.

1) Build the compiler with ``CHPL_TARGET_COMPILER`` set to ``gnu``:
    ```
    CHPL_TARGET_COMPILER=gnu make
    ```